tests/transactional/enable_selinux.pm

```diff
@@ -84,7 +83,7 @@ sub run {
     # see https://github.com/SELinuxProject/selinux-kernel/wiki/DEPRECATE-runtime-disable
     replace_grub_cmdline_settings('security=selinux selinux=1', 'security=selinux selinux=0', update_grub => 1);
     process_reboot(trigger => 1);
-    check_disabled;
+    selinuxtest::check_disabled;
```

Comment:
if you are importing an exported function by selinuxtest library, you do not need to add the namespace.

Reply:
When running the VR without the namespce it was [failing](http://mgrossu-openqa.qe.suse.de/tests/214) because of `Bareword "check_disabled" not allowed while "strict subs"` so I change to not importing `check_disabled`.

Reply:
Probably because it was not exported... now I see it exported in `lib/selinuxtest.pm`. So, for consistency, what I would do is:
```
use selinuxtest qw(check_disabled);
...
check_disabled;